commons/active/banking/banking_event.avsc

```diff
@@ -0,0 +1,23 @@
+{
+  "namespace": "org.radarcns.active.banking",
+  "type": "record",
+  "name": "BankingEvent",
+  "doc": "A Banking App Event, which is either a correct or wrong submission of a PIN number, an amount to withdraw, a cancelled or confirmed transaction. Notably, this allows us to distinguish between a wrong confirmed transaction or a correct cancelled transaction etc.",
+  "fields": [
+    { "name": "uid", "type": "string", "doc": "Unique ID for an Event."},
+    { "name": "time", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "timeCompleted", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "duration", "type": "double", "doc": "Duration of the activity (seconds)."},
```

Comment:
Is this a proxy for `duration = timeCompleted - time`? Perhaps this variable can be removed then.

Reply:
You mean remove duration variable? Maybe this is a useful variable to have for later purposes.  What do you think? I will gladly remove it otherwise.

Reply:
If it is simply the calculation of `duration = timeCompleted - time` it can be removed. If it is not, it may be described in a bit more detail what it is instead (screen load up to first button press? Now that I mention this, perhaps this should be stated a bit more explicitly in the time description as well. What does time and timeCompleted mean in your app? Time that the screen loaded up to time that a button was pressed?